sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryMetrics.java

```diff
@@ -864,10 +870,12 @@ private RntbdMetricsV2(MeterRegistry registry, RntbdTransportClient client, Rntb
                  .tags(tags)
                  .register(registry);
 
-            Gauge.builder(nameOf("rntbd.channels.acquired.count"), endpoint, RntbdEndpoint::channelsAcquiredMetric)
-                 .description("RNTBD acquired channel count")
-                 .tags(tags)
-                 .register(registry);
+            if (!ignoreMetricEnabled) {
+                Gauge.builder(nameOf("rntbd.channels.acquired.count"), endpoint, RntbdEndpoint::channelsAcquiredMetric)
```

Comment:
This should probably not be a Gauge - but a Counter or even summary - need to check - but what we would want to see is the total number of acquired/evicted channels in a  certain time period. Gauge would just be a snapshot ever interval - which wouldn't help for acquired/evicted

Reply:
I also am not sure whether this one should really be disabled - total count of acquired/evicted channels would be useful i believe - because if you see high values there over a certain period of time it is a clear indicator for connectivity issues.

Reply:
Agree with both comments. Counter or summary would work better. Will remove the if check.